src/pointer.rs

```diff
@@ -1,3 +1,5 @@
+#![allow(dead_code)]
```

Comment:
Why do we allow dead code here? If the code is no longer used, feel free to remove it.